test/test_art_span.cpp

```
@@ -0,0 +1,1116 @@
+/// tree.
+TYPED_TEST(ARTSpanCorrectnessTest, SingleKeyOperationsOnEmptyTree) {
+  const auto key = unodb::test::test_keys[1];  // 0x 00 02
+  const auto val = unodb::test::test_keys[2];  // 0x 03 00 01
+  TypeParam db;
+  EXPECT_FALSE(db.get(key));
+  {
+    uint64_t n = 0;
+    auto fn = [&n](const unodb::visitor<typename TypeParam::iterator>&) {
+      n++;           // LCOV_EXCL_LINE
+      return false;  // LCOV_EXCL_LINE
+    };
```

Cppcheck (2.10-2)

[error] There is an unknown macro here somewhere. Configuration is required. If UNODB_DETAIL_DISABLE_CLANG_WARNING is a macro then please configure it.

(unknownMacro)